gitstrap.go

```diff
@@ -274,14 +255,11 @@ func (strap *strapDestr) Run(opt Options) error {
 	if !prompt("you are going to remove Github repository and local git repository. Are you sure?") {
 		return nil
 	}
-	me, _, err := strap.base.cli.Users.Get(strap.base.ctx, "")
+	owner, err := getOwner(strap.base, opt)
 	if err != nil {
 		return strap.err("failed to get current user", err)
 	}
-	owner := *me.Login
-	if o, has := opt["org"]; has {
-		owner = o
-	}
+	// @todo #9 Extract name resolution to a separate function.
```

Comment:
@vassilevsky let's use single generic `todo` with all next steps in one (when creating a puzzle with single logical context)

Reply:
Improved the puzzle.